packages/preview/frogst/1.0.0/README.md

```diff
@@ -0,0 +1,20 @@
+# frogst
+
+A [typst](https://github.com/typst/typst) package for writing numbers in french.
+This package deals about the literal naming of numbers in French before the 1990 orthograph reform.
+
+## Example
+```typst
+#import "@preview/frogst:1.0.0": fr-nb
+#set page(width: 10cm, height: auto, margin: 0.7cm)
+#set par(justify: true)
+
+Le nombre 2713 s'écrit #fr-nb(2713).
+
+La vitesse de la lumière dans le vide est exactement #fr-nb(299792458) mètres par seconde.
+```
+![ouptut image of the code above](./examples/readme.png)
+
+## Future improvements
+- orthorapgh of 1990 reform variant
```

Comment:
I think "orthograph" is not really a word in English, I would suggest to use "spelling" instead. If I'm mistaken and "orthograph" does exist, there is at least a typo to fix (the g should be before the r, not between the p  and h).

```suggestion
- 1990 reform spelling variant
```

Reply:
Yeah sorry, my english is really terrible